### ISSUE TITLE
Module not found

### ISSUE THREAD

**Issue by zonzujiro** [gpbl/react-day-picker#340]
Hey!

Not sure, what's going on, but I have this error `Module not found: Error: Can't resolve 'react-day-picker/DayPickerInput'`  when using this example http://react-day-picker.js.org/examples/?simpleInput

Also, my webpack also shows me an error `Dynamic page loading failed Error: Cannot find module "react-day-picker/DayPickerInput"`

**Comment by gpbl** [comment]
@zonzujiro are you sure you are loading the last version?

```
$ npm ls react-day-picker
/react-day-picker/examples
└── react-day-picker@5.5.0
````

**Comment by zonzujiro** [comment]
Yes, I have 5.5.0. Installed with yarn

**Comment by gpbl** [comment]
Thanks, seeing it! Releasing a fix soon

**Comment by gpbl** [comment]
@zonzujiro this has been fixed in `v5.5.1` – please upgrade (and remember to add `moment` to your dependencies).
Thanks for the report!

**Comment by zonzujiro** [comment]
Wow, thanks! It was fast :)
